.github/pull_request_template.md

```diff
@@ -1,67 +1,126 @@
-  - Documentation update
-    - [ ] Done
-    - [ ] Not needed
-
-## Repository / Releases
-
-- [ ] Blocking PRs have been merged
-- [ ] Rebase / update of branch done
-- [ ] PR to `master` branch (Product Owner / Scrum Master)
-  - [ ] The [version](../dds_web/version.py) is updated
-    - [ ] I am bumping the major version (e.g. 1.x.x to 2.x.x)
-      - [ ] I have made the corresponding changes to the CLI version
-  - Backward compatible
-    - [ ] Yes: The code works together with `dds_cli/master` branch
-    - [ ] No: The code **does not** entirely / at all work together with the `dds_cli/master` branch. _Please add detailed and clear information about the broken features_
-
-## Checks
-
-- [ ] CodeQL passes
-- [ ] Formatting: Black & Prettier checks pass
-- Tests
-  - [ ] I have added tests for the new code
-  - [ ] The tests pass
-- Trivy / Snyk:
-  - [ ] There are no new security alerts
-  - [ ] This PR fixes new security alerts
-  - [ ] Security alerts have been dismissed
-  - [ ] PR will be merged with new security alerts; This is why: _Please add a short description here_
+## **5. Checklist**
+
+<!-- Comment out the items which do not apply here.-->
+
+### **Always**
+
+<!-- Always go through the following items. -->
+
+- [Changelog](../CHANGELOG.md)
+  - [ ] Added
+  - [ ] Not needed (E.g. PR contains _only_ tests)
+- Rebase / Update / Merge _from_ base branch
+  - [ ] Done
+  - [ ] Not needed
+- Blocking PRs
+  - [ ] Merged
+  - [ ] No blocking PRs
+- PR to `master` branch
+  - [ ] Yes: Go to the section [PR to master](#pr-to-master)
+  - [ ] No
+
+### If PR consists of **code change(s)**
+
+<!-- If the PR contains code changes, the following need to be checked.-->
+
+- Self review
+  - [ ] Done
+- Comments, docstrings, etc
+  - [ ] Added / Updated
+- Documentation
+  - [ ] Updated
+  - [ ] Update not needed
+
+### If PR is to **master**
+
+<!-- Is your PR to the master branch? The following items need to be checked off. -->
+
+- [ ] I have followed steps 1-5 in [the release instructions](../doc/procedures/new_release.md)
+- [ ] I am bumping the major version (e.g. 1.x.x to 2.x.x)
+- [ ] I have made the corresponding changes to the CLI version
+
+**Is this version _backward compatible?_**
+
+- [ ] Yes: The code works together with `dds_cli/master` branch
+- [ ] No: The code **does not** entirely / at all work together with the `dds_cli/master` branch. _Please add detailed and clear information about the broken features_
+
+## **6. Actions / Scans**
+
+<!-- Go through all checkboxes. All actions must pass before merging is allowed.-->
+
+- **Black**: Python code formatter. Does not execute. Only tests.
+  Run `black .` locally to execute formatting.
+  - [ ] Passed
+- **Prettier**: General code formatter. Our use case: MD and yaml mainly.
+  Run `npx prettier --write .` locally to execute formatting.
+  - [ ] Passed
+- **Tests**: Pytest to that verify functionality works as expected.
```

Comment:
maybe an extra _to_ here

Reply:
Yeah I think I meant "Pytest to verify [...]"